packages/functional-tests/src/tests/reparent-test.ts

```diff
@@ -45,7 +48,9 @@ export default class ReparentTest extends Test {
                     color: MRESDK.Color3.Yellow()
                 }
             }
-        });
+        }).value;
+        label.setBehavior(MRESDK.ButtonBehavior).onClick('released', () => this.running = false);
+
```

Comment:
is this label clickable wihtout having a collidable flag set anywhere? are colliders created by setbehavior, or automatically, or am I not reading this right?

Reply:
This is here for the future when colliders on empty objects with text are possible. Of course by that time, we'll probably have a generalized way to exit a test and it won't be necessary.

---
In reply to: [254026146](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/173#discussion_r254026146) [](ancestors = 254026146)